### PR TITLE
Application deployment

### PR DESCRIPTION
**Pull request by lordksix**
In this PR, the changes are as follows:
- Implementation of book, games, and music_albums classes
- Implementation of labels, genres, and authors classes
- Implement methods to save and load files with information on items created.
- Implement UI and classes to handle the creation of music albums, books, and games, and the listing of labels, authors, and genre
- Implement unit tests for books, games, music albums, labels, genres, and authors classes.
- Implement schema

### CODE REVIEW

File: games.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"games":"[{\"id\":335256,\"multiplayer\":2000,\"publish_date\":2000,\"archived\":false,\"last_played_at\":2022,\"author_id\":924870,\"first_name\":\"dgdfsgrfhn\",\"last_name\":\"tntrntr\"},{\"id\":981647,\"multiplayer\":2015,\"publish_date\":2015,\"archived\":false,\"last_played_at\":2023,\"author_id\":450941,\"first_name\":\"first\",\"last_name\":\"last\"},{\"id\":827194,\"multiplayer\":2021,\"publish_date\":2021,\"archived\":false,\"last_played_at\":2023,\"author_id\":969748,\"first_name\":\"nveonve\",\"last_name\":\"woebnfwbfn\"},{\"id\":497144,\"multiplayer\":2020,\"publish_date\":2020,\"archived\":false,\"last_played_at\":2002,\"author_id\":440569,\"first_name\":\"d\",\"last_name\":\"d\"}]"}
```
Comment (BregornOriginal) [inline games.json:1]: - [ ] Kindly fix the case when there's no `file.json` created and you try to run the app. It should create empty files to receive the new data. I'm doing references to books.json, games.json and music_albums.json

image

File: spec/game_spec.rb [hunk 2]
```diff
@@ -0,0 +1,65 @@
+require_relative '../src/game'
+require_relative '../src/author'
+
```
Comment (BregornOriginal) [inline spec/game_spec.rb:3]: - [ ] Kindly fix this test case

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Team 

Good job so far!

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


giphy (5)


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: games.json (hunk 1 above)
Comment (lordksix) [reply games.json:1]: Thank you for the feedback. The bug has been fixed.

File: spec/game_spec.rb (hunk 2 above)
Comment (lordksix) [reply spec/game_spec.rb:3]: Thank you for the feedback. The test has been fixed.

**Comment by tugsem** [comment]
## STATUS: APPROVED ✔️ 
Hi team, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Alt Text
Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**